src/plugins/discover/public/application/components/sidebar/discover_sidebar.tsx

```diff
@@ -108,16 +108,20 @@ export function DiscoverSidebar({
     setFields(newFields);
   }, [selectedIndexPattern, fieldCounts, hits, services]);
 
+  const uiCapabilites = getServices().capabilities.discover;
+
   const onShowDetails = useCallback(
     (show: boolean, field: IndexPatternField) => {
       if (!show) {
         setOpenFieldMap(new Map(openFieldMap.set(field.name, false)));
       } else {
         setOpenFieldMap(new Map(openFieldMap.set(field.name, true)));
-        selectedIndexPattern.popularizeField(field.name, 1);
+        if (uiCapabilites.save) {
```

Comment:
you can use services here
```suggestion
        if (services.capabilities.discover.save) {
```